tools/train.py

```diff
@@ -141,6 +141,8 @@ def main():
             mmaction_version=__version__ + get_git_hash(digits=7),
             config=cfg.text)
 
+    meta['config'] = cfg.pretty_text
```

Comment:
maybe it is more polite to sanitize paths in config.

since other repos clearly haven't concern privacy as much, let's merge it and fix it until someone actually raise an issue.

Reply:
what do u think, @dreamerlin , if u want to remove the folder structure, at least those contain user name